gradle/libs.versions.toml

```
@@ -16,6 +16,9 @@ kotlinx-bcv = "0.13.2"
 ## Analysis
 kotlin-compiler = "1.9.22"
 kotlin-compiler-k2 = "2.0.20-dev-7"
+# to run testing against the latest dev build of Analysis API
+# it is used in the build script of symbol analysis
+kotlin-compiler-latest = "2.0.20-dev-+"
```

Nitpick: 

The option name, `useTheLatestAnalysisAPI`, indicates the latest version should be used, but in the version it's only the latest 2.0.20 version.

What about keeping the name the same, but tell Gradle to use a dynamic version? Then it really will use the latest version.

```suggestion
kotlin-compiler-latest = "latest.integration"
```

---

We need the latest **dev** version.  the possible version list is [here](https://maven.pkg.jetbrains.space/kotlin/p/kotlin/kotlin-ide-plugin-dependencies/org/jetbrains/kotlin/analysis-api-standalone-for-ide/).
In some cases, the order of versions can be different from [Gradle one](https://docs.gradle.org/current/userguide/single_versions.html).
E.g. `1.7.0-RC-152` vs  `1.7.0-dev-1816`.

it seems I should rename the `kotlin-compiler-latest` property.

---

Yes, renaming it would be appreciated, thanks!